tests/test_node.py

```diff
@@ -1,12 +1,36 @@
+from unittest.mock import Mock
+
 import pytest
 
-from travertino.declaration import BaseStyle
+from tests.utils import mock_attr, prep_style_class
+from travertino.declaration import BaseStyle, Choices, validated_property
 from travertino.layout import BaseBox, Viewport
 from travertino.node import Node
 from travertino.size import BaseIntrinsicSize
 
 
+@prep_style_class
+@mock_attr("reapply")
 class Style(BaseStyle):
+    int_prop: int = validated_property(Choices(integer=True))
+
+    class IntrinsicSize(BaseIntrinsicSize):
+        pass
+
+    class Box(BaseBox):
+        pass
+
+    def layout(self, root, viewport):
+        # A simple layout scheme that allocates twice the viewport size.
+        root.layout.content_width = viewport.width * 2
+        root.layout.content_height = viewport.height * 2
+
+
+@prep_style_class
+class BrokenStyle(BaseStyle):
+    def reapply(self):
+        raise Exception
```

Comment:
I raised this previously; not sure why it was marked resolved. The issue is that this should be raising an *instance* of an exception, not an exception class:
```suggestion
        raise AttributeError("missing_attribute")
```

Reply:
Oh, my bad. I misinterpreted and thought you meant it was now irrelevant with the changes made to what was being tested.

I have no real objection, but I am curious — is there any reason to raise 1) a specific subtype of error and 2) an instance rather than class here? The behavior being tested should be catching any exception.

Reply:
(I don't think the previous conversation is marked as resolved, just outdated: https://github.com/beeware/travertino/pull/224#discussion_r1820097218 )

Reply:
Mostly to make sure test conditions accurately match what will be happening in production. You're correct that it should get caught regardless; but it's always best practice to match reality as much as possible to ensure that any edge cases are always caught, regardless of the implementation.

Reply:
That makes sense! Appreciate it.